index.js

```diff
@@ -40,6 +40,7 @@ const styles = StyleSheet.create({
     height: '100%',
     alignItems: 'center',
     justifyContent: 'space-around',
+    zIndex: -1
```

Comment:
Just to understand, why did you have to add `zIndex`? (a screenshot of what was not right without it?)

Reply:
Sure thing:
- Without zIndex:
![IMG_2711](https://user-images.githubusercontent.com/2279003/80869145-fef84580-8c74-11ea-9aa9-bdc7622a1b96.jpg)

- With zIndex:
![IMG_2713](https://user-images.githubusercontent.com/2279003/80869151-0ae40780-8c75-11ea-8775-99d1cee27e3f.jpg)

Reply:
ah.. nice

Reply:
We have been using it for a while like that in Android and iOS and it has been working fine, but my team upgraded my local version to react hooks, so it took me a while to make the diff and test it.

Reply:
@electhuang 

oh, right. We also have some pending items for this package, if you are willing, feel free to pick any of them.

Reply:
Let me ask @chescalante from my team, he already updated the component to hooks.

Reply:
Thanks, that would be awesome.